src/sympc/tensor/replicatedshare_tensor.py

```diff
@@ -209,6 +249,138 @@ def ne(self, y):
 
         """
 
+    @staticmethod
+    def __reconstruct_semi_honest(
+        share_ptrs: List["ReplicatedSharedTensor"],
+        get_shares: bool = False,
+    ) -> Union[torch.Tensor, List[torch.Tensor]]:
+        """Reconstruct value from shares.
+
+        Args:
+            share_ptrs (List[ReplicatedSharedTensor]): List of RSTensor pointers.
+            get_shares (bool): Retrieve only shares.
+
+        Returns:
+            reconstructed_value (torch.Tensor): Reconstructed value.
```

Comment:
I think return should also include `shares :List[torch.Tensor]`

Reply:
Hmmm, I followed the convention of MPCTensor where it was just reconstructed value, so thought of sticking to that.

Reply:
But, yeah I will modify it.

Reply:
No, I won't because it ends up making docstring too long giving lint errors. Let's keep it that way for now.